### PR TITLE
feat: highlight editable region

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
This gives the editable region a lighter background than the rest of the editor.  Hopefully this will draw the user's eye to the right part of the page.

Before:

BeforeDay
BeforeNight

and after:

AfterDay
AfterNight

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]
TBH I really love this.

**Comment by ojeytonwilliams** [comment]
@nhcarrigan I forget if it was @ahmadabdolsaheb's idea, but he's the one that realised it could be done so simply.

File: client/src/templates/Challenges/classic/editor.tsx [hunk 1]
```diff
@@ -259,10 +261,11 @@ const Editor = (props: EditorProps): JSX.Element => {
   };
 
   const editorWillMount = (monaco: typeof monacoEditor) => {
-    const { challengeFiles, fileKey } = props;
+    const { challengeFiles, fileKey, usesMultifileEditor } = props;
+    console.log('editorWillMount', props);
```
Comment (ShaunSHamilton) [inline client/src/templates/Challenges/classic/editor.tsx:265]:
```suggestion
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Personally, i am not a fan of this border (against the new background):
image

Also, changing the theme in editor results in the background not showing:
image

A refresh also does not yield any background for the light theme?

File: client/src/templates/Challenges/classic/editor.tsx (hunk 1 above)
Comment (ShaunSHamilton) [reply client/src/templates/Challenges/classic/editor.tsx:265]: `usesMultifileEditor` is always `false`, from what I can see?

**Comment by ojeytonwilliams** [comment]
> Also, changing the theme in editor results in the background not showing:

What steps should I take to see this?

The border is helpful, imo, since it helps you see where you're working. That said, we at least want to adjust the style @ahmadabdolsaheb :)

**Comment by ShaunSHamilton** [comment]
> What steps should I take to see this?

What I did was:
1) Develop client
2) Open `/learn`
3) Change theme to prevent permanent eye damage
4) Navigate to an UPCOMING challenge
5) Change theme to cause permanent eye damage
6) See there is no highlight

**Comment by ojeytonwilliams** [comment]
@ShaunSHamilton I followed your ophthalmic instructions and got this:

image

which looks right to me.

**Review by ShaunSHamilton — APPROVED** [review]
After doing something, the highlighting appeared for the Light theme. I do not know what changed 🤷‍♂️

**Review by naomi-lgbt — APPROVED** [review]
While I miss the poor console log that @ShaunSHamilton yeeted, I'm still happy with this. 🙂

**Comment by ahmadabdolsaheb** [comment]
Great functionality @ojeytonwilliams. I like how the editor UI in maturing. We could re-adjust the contrast around the editable region and play around with the borders to see if it would look better, but that should be straight forward.

**Pull request merged**